openff/interchange/interop/openmm/_topology.py

```diff
@@ -173,6 +186,21 @@ def to_openmm_topology(
                 order=bond_order,
             )
 
+    if has_virtual_sites and not collate:
+        virtual_site_chain = openmm_topology.addChain(atom_chain_id)
+        virtual_site_residue = openmm_topology.addResidue("VS", virtual_site_chain)
```

Comment:
From a practical standpoint, a reference to the original residue would possibly be useful - i.e. when I'm doing "solvent within N of Y" I'm looking for a residue name that matches SOL | WAT | HOH not VS.

Reply:
I think this would be solved by your suggestion later - that each "virtual site residue" be named according to the molecule/residue it's from?